Library/Homebrew/formula_installer.rb

```diff
@@ -263,7 +263,7 @@ def check_install_sanity
           You can try to install from source with:
             brew install --build-from-source #{formula}
           Please note building from source is unsupported. You will encounter build
-          failures with some formulae. If you experience any issues please create pull
+          failures with some formulas. If you experience any issues please create pull
```

Comment:
["formulae" is plural for "formula"](https://www.merriam-webster.com/dictionary/formula).

"formulas" works for modern usage too, but we should probably be consistent about this. There are 205 files in `brew` using "formulae" and no files using "formulas":
```
❯ rg 'formulae' -l $(brew --repo) | wc -l
205
❯ rg 'formulas' -l $(brew --repo) | wc -l
0
```
If we want to change this, we should change all other instances too. That said, I do have a small personal preference for "formulae" over "formulas".